### ISSUE TITLE
The Dantotsu "Add To List" feature fails to synchronize with MAL after an item has been removed from the list, whereas it successfully deletes items from Anilist without any issues.

### ISSUE THREAD

**Issue by rajinstar** [rebelonion/Dantotsu#480]
When I add an anime to my list and subsequently remove it, the entry is automatically deleted from Anilist but not from MyAnimeList. This creates additional work for me, as I then have to manually remove the anime from MyAnimeList. While adding an anime update to both Anilist and MyAnimeList, deleting an anime only reflects on Anilist. Is there a way to resolve this issue? 😕

I will provide you some Screenshots so that you can understand it properly!

![Screenshot_2024-09-17-02-08-26-920_ani.dantotsu.beta.jpg](https://github.com/user-attachments/assets/1ddb194b-9292-40b9-a476-62b878f54b27)

![Screenshot_2024-09-17-02-08-30-480_ani.dantotsu.beta.jpg](https://github.com/user-attachments/assets/430e2b6d-0c8b-4110-af49-1ac604ca7240)

![Screenshot_2024-09-17-02-08-36-348_ani.dantotsu.beta.jpg](https://github.com/user-attachments/assets/3a8668bf-cfbe-4555-af3e-02611fd17156)

![IMG_20240917_020932.jpg](https://github.com/user-attachments/assets/8f5fdd3c-ef83-4096-a43e-bcd12dd3f94f)

![IMG_20240917_020947.jpg](https://github.com/user-attachments/assets/04be32ba-b1f1-4644-93b5-644c3f3a5d82)

![IMG_20240917_021043.jpg](https://github.com/user-attachments/assets/e1372336-d5c2-449e-b762-be081ea754d8)

![Screenshot_2024-09-17-02-10-50-618_ani.dantotsu.beta.jpg](https://github.com/user-attachments/assets/a6a5772e-006d-44dd-a821-a1ff04bd507b)

![Screenshot_2024-09-17-02-12-15-552_ani.dantotsu.beta.jpg](https://github.com/user-attachments/assets/88fd77f0-0480-42ea-b376-af1987b639ea)

![Screenshot_2024-09-17-02-12-19-518_com.zen.alchan.jpg](https://github.com/user-attachments/assets/06deb036-f839-4172-abb6-d31727416bc3)

![Screenshot_2024-09-17-02-12-28-231_net.myanimelist.app.jpg](https://github.com/user-attachments/assets/5aac7628-4054-4699-ba75-93106958f683)

![Screenshot_2024-09-17-02-12-33-502_net.myanimelist.app.jpg](https://github.com/user-attachments/assets/4c23798e-1022-4dfd-a5c8-17d19c5fa76f)

![Screenshot_2024-09-17-02-12-39-369_net.myanimelist.app.jpg](https://github.com/user-attachments/assets/b64eca3c-f060-4899-86e0-819e2e3e03ce)

![Screenshot_2024-09-17-02-12-46-120_net.myanimelist.app.jpg](https://github.com/user-attachments/assets/c4c04902-586c-4f8a-ad0e-3fc184fc580a)

**Comment by grayankit** [comment]
Dantotsu is an anilist client so don't have an active dev on mal

**Comment by FiddleMySticks** [comment]
Yep, same thing here. This has been an issue for a long time. On version 2.2.0 of the app, this is not an issue, but after it was updated to 3.0.0 and beyond, removing items from the list just does not sync with MAL.

**Comment by itsmechinmoy** [comment]
Fixed by toby

@rebelonion close it
